main.go

```diff
@@ -219,13 +273,23 @@ func main() {
 	if filePth == "" {
 		filePth = cfg.PkgPath
 	}
+	if filePth == "" {
+		failf("Either IPA path or PKG path has to be provided")
+	}
 
 	additionalParams, err := shellquote.Split(cfg.AdditionalParams)
 	if err != nil {
 		failf("Failed to parse additional parameters, error: %s", err)
 	}
 
-	altoolParams := append([]string{"altool", "--upload-app", "-f", filePth}, authParams...)
+	uploadParams := []string{"--upload-app", "-f", filePth}
+	// Platform type parameter was introduced in Xcode 13
+	if xcodeVersion.MajorVersion >= 13 && !sliceutil.IsStringInSlice(typeKey, additionalParams) {
+		uploadParams = append(uploadParams, typeKey, getPlatformType(cfg.IpaPath, cfg.Platform))
```

Comment:
```suggestion
		uploadParams = append(uploadParams, typeKey, getPlatformType(filePth, cfg.Platform))
```

Reply:
It is intended that I pass only the ipa path. If it is not ipa (pkg) I pass "" and it is handled in the function